about.html

```diff
@@ -7,67 +7,95 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
 <body>
-    <div class="container about-page">
+  <header>
+  <nav>
+  <div class="lambdaLogo">
+    <img class ="logo"src="https://github.com/Lfritze/Sprint-Challenge--User-Interface/blob/master/img/lambda-black.png?raw=true" alt="Lambda Logo">
+  </div>
+  
+    <a href="index.html">Home</a>
+    <a href="#about">About</a>
+    <a href="#products">Products</a>
+    <a href="#blog">Blog</a>
+    <a href="#contact">Contact</a>
+    </nav>
+  </header>
+  
+    <div class="container">
+      
+      <div class="top-img">
+        <img class="top-picture"src="https://github.com/Lfritze/Sprint-Challenge--User-Interface/blob/master/img/jumbo-about.png?raw=true" alt="city picture">
+        </div>
+
+      <section class="top-content">
+        <div class ="text-container">
      
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
-    
-    Places We Work
-
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    <img src="img/about-meeting.png" alt="strategy">
-    
-    Collaboration
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          <h2>
+            About
+          </h2>
+
+            <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+
+      </section>
+
+      <section class="middle">
```

Comment:
what properties would you need to add to make the concent appear to be 'pinched in' from the edges of the container?